services-directions-models/src/main/java/com/mapbox/api/directions/v5/models/StepIntersection.java

```diff
@@ -480,6 +502,28 @@ public abstract static class Builder extends DirectionsJsonObject.Builder<Builde
     @NonNull
     public abstract Builder yieldSign(@Nullable Boolean yieldSign);
 
+    /**
+     * An object containing information about routing and passing IC(s) along the route.
+     *
+     * @param interChange object containing information about routing and passing IC(s)
+     *   along the route.
+     * @return an object containing information about routing and passing IC(s) along the
```

Comment:
It returns a builder, not "an object containing information about routing and passing IC(s)..."